spec/lib/castle/client_spec.rb

```diff
@@ -167,19 +167,19 @@
       let(:options) { { event: '$login.succeeded', user_id: '1234' } }
 
       it do
-        assert_requested :post, 'https://api.castle.io/v1/track',
-                        times: 1,
-                        body: request_body.to_json
+        assert_requested :post, 'https://api.castle.io/v1/track', times: 1 do |req|
+          JSON.parse(req.body) == JSON.parse(request_body.to_json)
+        end
       end
     end
 
     context 'string keys' do
       let(:options) { { 'event' => '$login.succeeded', 'user_id' => '1234' } }
 
       it do
-        assert_requested :post, 'https://api.castle.io/v1/track',
-                        times: 1,
-                        body: request_body.to_json
+        assert_requested :post, 'https://api.castle.io/v1/track', times: 1 do |req|
+          JSON.parse(req.body) == JSON.parse(request_body.to_json)
```

Comment:
@nijikon is there a better way to test this. The order has changed after the code change so String comparison fails